### PR TITLE
Add script to check SDL does not resend OnSeekMediaClockTimer if seek…

### PR DESCRIPTION
**Pull request by OVikhrov**
…Time hours out lower bound
@aarotar Please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Lower_Bound.lua [hunk 1]
```diff
@@ -0,0 +1,94 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [MOBILE_API] New OnSeekMediaClockTimer notification
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Lower_Bound.lua]: @OVikhrov, please update Requirement summary with related **functional** requirement also

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Lower_Bound.lua [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+-- [HMI_API] <param name="hours" type="Integer" minvalue="0" maxvalue="59" mandatory="true">
+-- [MOBILE_API] <param name="hours" type="Integer" minvalue="0" maxvalue="59">
+--
+-- 1. Used preconditions:
+-- a) First SDL life cycle
+-- b) OnSeekMediaClockTimer notification allowed in preloaded file for default app
+-- c) App successfylly registered, consented and activated
+--
+-- 2. Performed steps:
+-- a) HMI sends OnSeekMediaClockTimer notification to SDL with hours put lower bound value ("-1") of seekTime parameter.
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Lower_Bound.lua]: @OVikhrov, please correct step a description with "..**out** lower bound..." Currently it specifies lower bound value

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Lower_Bound.lua [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+function Test:Preconditions_ActivateApp()
+commonSteps:ActivateAppInSpecificLevel(self, self.applications["Test Application"])
+EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_OnSeekMediaClockTimer_hours_Value_Of_seekTime_Is_Out_Lower_Bound()
+ self.hmiConnection:SendNotification("UI.OnSeekMediaClockTimer",{seekTime =  {hours = -1, minutes = 1, seconds = 33}, appID = self.applications["Test Application"]})
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Lower_Bound.lua]: @OVikhrov, plase avoid redundant empty lines

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**